### PR TITLE
igor: Add web interface to igor (igorweb)

### PR DESCRIPTION
**Pull request by mwisely**
`igorweb` allows users to create/modify/delete `igor` reservations using a friendly web interface. It has all of the functionality of `igor` in a shiny package. 

`igorweb` is its own package separate from `igor` and issues commands to `igor`. So, `igor` works without `igorweb`, but `igorweb` can't work without `igor`.

### CODE REVIEW

File: src/igorweb/show.go [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+// GetGroup returns the user.Group that can edit tihs
+// reservation. Returns nil if there's no group set
+func (r Res) GetGroup() *user.Group {
+	if r.Group == "" {
+		return nil
+	}
+
+	g, err := user.LookupGroup(r.Group)
+	if err != nil {
+		log.Warn("Unable to lookup group \"s\": %v", r.Group, err)
```
Comment (djfritz) [inline src/igorweb/show.go]: This looks to be missing a % in the format string. Perhaps:

```suggestion
		log.Warn("Unable to lookup group \"%s\": %v", r.Group, err)
```

**Comment by djfritz** [comment]
@csymonds @mwisely

**Comment by djfritz** [comment]
Bump @mwisely @csymonds

**Comment by djfritz** [comment]
Where did the travis integration go on this?

**Comment by djfritz** [comment]
@csymonds can I merge this?

**Comment by csymonds** [comment]
@djfritz We are good on our end.

**Pull request merged**